src/improvements/tasks/example/sep/002-opcm-upgrade-v200/config.toml

```diff
@@ -0,0 +1,10 @@
+l2chains = [{name = "OP Sepolia Testnet", chainId = 11155420}]
+
+templateName = "OPCMUpgradeV200"
+
+[opcmUpgrades]
+absolutePrestates = [{absolutePrestate = "0x035ac388b5cb22acf52a2063cfde108d09b1888655d21f02f595f9c3ea6cbdcd", chainId = 11155420}]
+standardValidatorV200 = "0x37739a6b0a3F1E7429499a4eC4A0685439Daff5C"
+
+[addresses]
+OPCM = "0x1B25F566336F47BC5E0036D66E142237DcF4640b"
```

Comment:
I think a good practice would be using code comments to attest to the various values here, like links/explanations to how we know this prestate is good, and somewhere that shows these `standardValidatorV200` and `OPCM` addresses are correct

Reply:
Addressed in: https://github.com/ethereum-optimism/superchain-ops/pull/664/commits/c908437b9e807b01b16c9867d5f9b15aa62d615b

For prestates, I'll make a note to add comments in the actual task configs.